### ISSUE TITLE
Typing brackets in Elements List causes Errors

### ISSUE THREAD

**Issue by fisher729** [nvaccess/nvda#8680]
### Steps to reproduce:
1. Open the Elements List
2. Press the brackets on the keyboard.
### Actual behavior:
Error tone is played
### Expected behavior:
No error tone is to be played
### System configuration:
Windows 10 1803 64-bit
#### NVDA Installed/portable/running from source:
Installed
#### NVDA version:
2018.3beta3
#### Name and version of other software in use when reproducing the issue:
Firefox 61.0.2.

#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA?
No

I wonder if the Elements List is being overlooked these days. I reported an issue with the list whereby another tone is heard. There are also instances where the list cannot be closed. I think this needs more attention.

**Comment by jscholes** [comment]
I can't reproduce this issue.  Can you post a log so we can see the error?

**Comment by fisher729** [comment]
Try using the square brackets
null

**Comment by jscholes** [comment]
I've tried parentheses, brackets and braces, and none of them cause an error sound to be played.  So we're going to need a log in order to determine what's going wrong.  Please reproduce the issue and then post a log file here.  You can find the log file by opening the Run dialog, typing %temp% and pressing Enter, and then locating the file nvda.log.  Alternatively you can post text from the NVDA Log Viewer, which can be opened with NVDA+F1.

**Comment by fisher729** [comment]
Hi.

This error occurs in the Filter by field. I should've mentionned that
from the start.

ERROR - unhandled exception (10:00:24.736):
Traceback (most recent call last):
  File "browseMode.pyo", line 1025, in onFilterEditTextChange
  File "browseMode.pyo", line 890, in filter
wxAssertionError: C++ assertion "item.IsOk()" failed at
..\..\src\msw\treectrl.cpp(1032) in wxTreeCtrl::GetItemParam():
invalid tree item

**Comment by bhavyashah** [comment]
Confirmed. I have tested this very briefly, but using Firefox 61.0.2 and NVDA 2018.3 Beta 3 on https://en.wikipedia.org/wiki/India, I can generate error sounds with all three brackets. I have not tested any other pages (that may contain less elements), characters (which may also produce this behaviour and make this issue non-exclusive to brackets only), or browser. Since a log fragment has already been provided in https://github.com/nvaccess/nvda/issues/8680#issuecomment-416256765, I will provide my own if called for.
